rfq/manager_test.go

```diff
@@ -0,0 +1,352 @@
+package rfq
+
+import (
+	"context"
+	"encoding/binary"
+	"testing"
+
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
+	"github.com/lightninglabs/lndclient"
+	"github.com/lightninglabs/taproot-assets/address"
+	"github.com/lightninglabs/taproot-assets/asset"
+	"github.com/lightninglabs/taproot-assets/proof"
+	tpchmsg "github.com/lightninglabs/taproot-assets/tapchannelmsg"
+	"github.com/lightningnetwork/lnd/lnwallet"
+	"github.com/lightningnetwork/lnd/routing/route"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	asset1 = asset.Asset{
+		ScriptKey: asset.ScriptKey{
+			PubKey: asset.NUMSPubKey,
+		},
+		Genesis: asset.Genesis{
+			Tag: "111",
+		},
+	}
+	asset2 = asset.Asset{
+		ScriptKey: asset.ScriptKey{
+			PubKey: asset.NUMSPubKey,
+		},
+		Genesis: asset.Genesis{
+			Tag: "222",
+		},
+	}
+	asset3 = asset.Asset{
+		ScriptKey: asset.ScriptKey{
+			PubKey: asset.NUMSPubKey,
+		},
+		Genesis: asset.Genesis{
+			Tag: "333",
+		},
+	}
+	testAssetID1 = asset1.ID()
+	testAssetID2 = asset2.ID()
+	testAssetID3 = asset3.ID()
+	proof1       = proof.Proof{
+		Asset: asset1,
+	}
+	proof2 = proof.Proof{
+		Asset: asset2,
+	}
+	proof3 = proof.Proof{
+		Asset: asset3,
+	}
+	testGroupKey = pubKeyFromUint64(2121)
```

Comment:
Perhaps we should just set this to a random group key? I think we have functions for generating random group keys. And then we don't need `pubKeyFromUint64`. Although, I do like `pubKeyFromUint64`, random key is better practice for more coverage IMO.